src/Microsoft.VisualStudio.Validation/Requires.cs

```diff
@@ -70,7 +70,7 @@ public static IntPtr NotNull(IntPtr value, string parameterName)
         /// </remarks>
         [DebuggerStepThrough]
         [TargetedPatchingOptOut("Performance critical to inline across NGen image boundaries")]
-        public static void NotNull([ValidatedNotNull]Task value, string parameterName)
+        public static void NotNull([ValidatedNotNull, NotNull]Task value, string parameterName)
```

Comment:
I think the type of value should be nullable `Task?`. Oh, should enable `Nullable` feature first.

Reply:
This code base still compiles with C# compiler version 7.3.

Reply:
The consequence of this is that no `NullableAttribute` nor `NullableContextAttribute` attributes are included in compiled binaries. However it's still possible to annotate with pre/post-conditions.

Note that if a newer compiler is used and NRT is enabled, lots of these methods will need to be annotated to allow nullable values. For example `NotNull(Task<T?>?)` needs it's type parameter annotated to prevent callers having errors. The same is true for collection verification methods.

Reply:
Yes, I know.

Reply:
We'll save this for a subsequent change.